docs/data/data-grid/recipes-editing/recipes-editing.md

```diff
@@ -105,18 +105,3 @@ Using the [controlled mode](/x/react-data-grid/editing/#controlled-model) and li
 The following demo implements this behavior.
 
 {{"demo": "SingleClickEditing.js", "bg": "inline", "defaultCodeOpen": false}}
-
```

Comment:
Moved to https://deploy-preview-9228--material-ui-x.netlify.app/x/react-data-grid/custom-columns/